zap/src/main/java/org/parosproxy/paros/control/MenuFileControl.java

```diff
@@ -595,8 +596,13 @@ public String getDescription() {
                     return;
                 }
                 // Import the context
-                Model.getSingleton().getSession().importContext(file);
+                Context ctx = Model.getSingleton().getSession().importContext(file);
```

Comment:
Extract a method to create the context and then check if the name already exists (and ask the user for the new name) before importing.

Reply:
Do you mean in `Session` I should create a new method that creates a `Context` without checking the name? (Or setting it into the Session I guess.)